### ISSUE TITLE
I'm Having trouble finding the alterative fonts

### ISSUE THREAD

**Issue by nosipitan** [Etcetera-Type-Co/Grandstander#5]
I'm Having trouble finding the alterative fonts that includes the other version of the letters (J V Y a l r t y) in the fonts folders. Could you please help me?

**Comment by sursly** [comment]
They are stylistic alternates that you need to select when using the font, not separate files. You can find this as an opentype option in many applications, or on the web as ` font-feature-settings: 'ss02`

**Comment by nosipitan** [comment]
> They are stylistic alternates that you need to select when using the font, not separate files. You can find this as an opentype option in many applications, or on the web as ` font-feature-settings: 'ss02`

I decided to open your font in the 'Character Map' application that comes with Windows based on my understanding of what you said, but I still couldn't find the alternate letters I mentioned earlier. Is there another way I should be looking for these, or do I need to use a different tool?

**Comment by sursly** [comment]
@nosipitan it's possible that your download doesn't have it. Did you get the latest release from here on Github? (I don't believe the version from google fonts is up-to-date).

**Comment by nosipitan** [comment]
> @nosipitan it's possible that your download doesn't have it. Did you get the latest release from here on Github? (I don't believe the version from google fonts is up-to-date).

Yes, I downloaded it from GitHub and resolved the issue by opening the font in "FontForge." I switched the alternate versions with the standard ones, so now I can use the font without needing to go through apps like Adobe Illustrator etc. Your advice about OpenType and stylistic alternates was really helpful, as I didn't even know the source of the problem before. Thanks!
